xpdacq/xpdacq.py

```diff
@@ -622,9 +622,6 @@ def __call__(self, sample, plan, subs=None, *,
                                'global metadata')
         if robot:
             metadata_kw.update(robot=True)
-        if not robot and isinstance(sample, list):
```

Comment:
why was this even in?  Did we put it there for some expectation management or sthg?

Reply:
Yes I think we weren't going to get the chance to test this so we didn't want people doing it by accident.

Reply:
gotcha

Reply:
@CJ-Wright do you want this pulled in?

Reply:
yes please!